src/components/Terminal/ASABTerminal.jsx

```diff
@@ -0,0 +1,216 @@
+import React, { useEffect, useRef } from 'react';
+import { Card, CardHeader, CardBody } from 'reactstrap';
+// import { useSelector } from 'react-redux';
+
+import { Terminal } from '@xterm/xterm';  // Use @xterm/xterm package
+import { FitAddon } from '@xterm/addon-fit';  // Import the FitAddon
+import '@xterm/xterm/css/xterm.css'; // Import the xterm.css
+
+/*
+
+	TODO:
+
+	- write README with usage (for websocket and for regular get API calls)
+	- add styling based on theme
+	- play with other setting
+	- extend by option to write (and send) commands within the terminal (in future iterations)
```

Comment:
TODO: add pubsub for `Terminal.reload!` for reloading the terminal content within the loader (similarly as it is in the DataTable2)